tests/core/test_views_version.py

```diff
@@ -1,30 +1,42 @@
-from kinto.core.views import version
-from kinto.core.testing import unittest
+import mock
+import os
+import json
+import tempfile
+import unittest
 
+from kinto.core.views.version import version_view
 from .support import BaseWebTest
 
 
 class VersionViewTest(BaseWebTest, unittest.TestCase):
-
-    def setUp(self):
-        self.VERSION_JSON_before = version.VERSION_JSON
-        self.ORIGIN_before = version.ORIGIN
-
     def tearDown(self):
-        version.VERSION_JSON = self.VERSION_JSON_before
-        version.ORIGIN = self.ORIGIN_before
-
-    def test_return_the_version_file_if_present(self):
-        version.VERSION_JSON = {
-            "commit": "ce480821debc080d9cf5cb67e9a6ba71b2c3f57e",
-            "version": "0.8.1",
-            "name": "kinto-dist",
-            "source": "https://github.com/mozilla-services/kinto-dist"
-        }
-        response = self.app.get('/__version__')
+        try:
+            os.remove('version.json')  # Clean-up file at default location.
+        except OSError:
+            pass
+        try:
+            delattr(version_view, '__json__')  # Clean-up memoization.
+        except AttributeError:
+            pass
+
+    def test_return_the_version_file_in_current_folder_if_present(self):
+        with open('version.json', 'w') as f:
+            json.dump({'version': '0.8.1'}, f)
 
-        assert response.json == version.VERSION_JSON
+        response = self.app.get('/__version__')
+        assert 'version' in response.json
```

Comment:
Maybe test the full response?
```assert response.json == "{'version': '0.8.1'}"```
This way, if we have some other code in the future that overrides the version, this test will fail ;)